Designer/View/GeneralPopup.xaml

```diff
@@ -1,16 +1,19 @@
-﻿<Window x:Class="Designer.View.RoomEditorPopupView"
+﻿<Window x:Class="Designer.View.GeneralPopup"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Designer.View"
         mc:Ignorable="d"
-        Title="RoomEditorPopupView" Height="200" Width="400">
+        Title="Popup" Height="200" Width="400">
```

Comment:
Wellicht hier een binding gebruiken

Reply:
Het is mij niet gelukt dit te laten werken met een binding, heb jij een manier?